FitGoal/FitGoal/HomeViewController/HomeViewController.swift

```diff
@@ -142,3 +145,11 @@ extension HomeViewController {
         }
     }
 }
+
+extension HomeViewController: RoutineInspectorCellDelegate {
+    func userDidSelectExersice(_ exercise: Exercise) {
+        let vc = ExerciseDetailViewController()
+        vc.exercise = exercise
```

Comment:
It should be better if you pass the exercise in the parameter of the view controller instead of setting it as a field. That way you can get rid of the optional `exercise` field in the inside the `ExerciseDetailViewController`.